### PR TITLE
About page

### PR DESCRIPTION
**Pull request by PJSalter**
- Created about me section.
- included text about the agency, illustration and layout for all screens.
- illustration about what we do as an agency.
- figcaption tag with some description.
- included image of myself for employees section page.
- included image of Logo.
- altered styling for services page for responsiveness.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,9 +34,28 @@ <h1 class="position-text">Don't let your business be left behind. <br>
...
+                VED has brocken all barriers by keeping up to date with it's services, providing the best and most 
+                prominent industry leading technology that is developed by our high skilled software developers.<br><br>
+                Several Businesses from all over the global have reached out to us because of word of mouth from our
+                spectacular services. We have worked with huge companies such as Adobe and even Sony to name some of
+                the larger companies. <br><br>
+                Excitingly we are expanding our team to bring the best and the brightest and to ensure that quick services
+                and proficiency is in order, we care about our customers to the point that they will get amazing designs faster. <br><br>
+                Now currently celebrating our 4th year since opening are doors, we have consitently won many awards for best 
+                practices from a web development agency, and many more nominations for fast and effective outcomes. <br><br>
+                We deliver the impossible and throw out many smiles.
```
Comment (sonianb) [inline index.html:56]: Wow, we're an awesome agency! I'm proud of us!

File: index.html [hunk 2]
```diff
@@ -34,9 +34,28 @@ <h1 class="position-text">Don't let your business be left behind. <br>
             Scale up with VED.
             </h1>
         </section>
-        <section id="about">
-            <h2>About</h2>
 
+        <section id="about">
+            <h1 class="about-title center">About Us</h1>
```
Comment (sonianb) [inline index.html]: We're already using h1 in the hero section of the page, so we'd have to make this a h2.

File: styles.css [hunk 3]
```diff
@@ -169,24 +169,22 @@ body {
 /* Services Section */
 
 h1{
```
Comment (sonianb) [inline styles.css:208]: I'll create a class for that because it'll change the style for the existing h1.

File: styles.css [hunk 4]
```diff
@@ -324,3 +353,128 @@ h2, h3 {
...
+    height: auto;
+    /* padding: 1.5rem; */
+}
+
+.image-description{
+    text-align:center;
+    color: var(--primary-color);
+}
+
+/* ------------- Media Queries for about Page --------------------------- */
```
Comment (sonianb) [inline styles.css]: Hmmm wouldn't it be better to put all the media queries for all the pages at the end?

File: styles.css [hunk 5]
```diff
@@ -324,3 +353,128 @@ h2, h3 {
...
+
+@media only screen and (max-width: 1440px) {
```
Comment (sonianb) [inline styles.css]: You have already defined this breakpoint at line 269, so it should be moved there.

File: styles.css [hunk 6]
```diff
@@ -324,3 +353,128 @@ h2, h3 {
...
+    .about-text-styling{
+    text-align: right;
+    font-size: 1.4rem;
+    }
+    .image-description{
+    font-size: 1.6rem;
+    }
+}
+
+@media only screen and (max-width: 1080px) {
```
Comment (sonianb) [inline styles.css]: Same here, this breakpoint already exists in line 289.

**Review by sonianb — CHANGES_REQUESTED** [review]
One breakpoint for all the sections of the page. 
This should be a quick fix - just copy and paste the code in the correct breakpoint.

File: index.html (hunk 1 above)
Comment (PJSalter) [reply index.html:56]: 🌟

File: index.html (hunk 2 above)
Comment (PJSalter) [reply index.html]: Ok i'll resolve this with a h2 😄

File: styles.css (hunk 3 above)
Comment (PJSalter) [reply styles.css:208]: Thank you ✨

File: styles.css (hunk 4 above)
Comment (PJSalter) [reply styles.css]: It would be a lot neater and easier to read, i'll adjust this now, thats a very good point ⭐

File: styles.css (hunk 5 above)
Comment (PJSalter) [reply styles.css]: Just resolved this 😄

File: styles.css (hunk 6 above)
Comment (PJSalter) [reply styles.css]: Thank you for letting me know

**Comment by sonianb** [comment]
Closes #10 #12

**Pull request merged**